libkbfs/tlf_journal.go

```diff
@@ -813,6 +826,56 @@ func (e errTLFJournalNotEmpty) Error() string {
 	return "tlfJournal is not empty"
 }
 
+func (j *tlfJournal) checkServerForConflicts(ctx context.Context) error {
+	if time.Since(j.lastServerMDCheck) < tlfJournalServerMDCheckInterval ||
+		!j.config.MDServer().IsConnected() {
```

Comment:
If we are offline, one of the flush operations will block or timeout, leading to an exponential backoff retry at a higher level.  I only put this check here to avoid blocking on a pretty useless command, but it's not strictly necessary and I don't think there's a reason to explicitly reconnect here and not do so everywhere else there's an RPC.

I think I'll remove the check, it's just confusing.